print-mptcp.c

```diff
@@ -454,6 +480,23 @@ mp_fast_close_print(netdissect_options *ndo,
         return 1;
 }
 
+static int
+mp_tcprst_print(netdissect_options *ndo,
+                const u_char *opt, u_int opt_len, u_char flags _U_)
+{
+        const struct mp_tcprst *mpr = (const struct mp_tcprst *)opt;
+
+        if (opt_len != 4)
+                return 0;
+
+        ND_PRINT(" flags [%s]", bittok2str_nosep(mp_tcprst_flags, "none",
+                 GET_U_1(mpr->sub_b)));
+
+        ND_PRINT(" reason %s", tok2str(mp_tcprst_reason, "invalid (0x%02x)",
```

Comment:
detail: should we indicate `unknown` instead of `invalid`? The RFC specifies 7 possible reasons but it doesn't forbid extensions later.

https://www.rfc-editor.org/rfc/rfc8684.html#name-subflow-reset

Reply:
indeed, it's not `invalid ` just because it's not a known value of `mp_tcprst_flags`. Will fix that.